pkg/gofr/crud_handlers.go

```diff
@@ -117,10 +119,12 @@ func (e *entity) Create(c *Context) (interface{}, error) {
 		fieldValues = append(fieldValues, reflect.ValueOf(newEntity).Elem().Field(i).Interface())
 	}
 
-	stmt := fmt.Sprintf("INSERT INTO %s (%s) VALUES (%s)",
-		e.name,
-		strings.Join(fieldNames, ", "),
-		strings.Repeat("?, ", len(fieldNames)-1)+"?",
+	stmt := sql.Rebind(c.SQL.Dialect(),
+		fmt.Sprintf("INSERT INTO %s (%s) VALUES (%s)",
```

Comment:
I am getting the following error when I am trying to test this feature - 
`{
    "error": {
        "message": "pq: syntax error at or near \"user\""
    }
}`

Similar error for getByID
`
{
    "error": {
        "message": "pq: column \"id\" does not exist"
    }
}
`

Need to check all of them, probably a slight syntax change for postgres and sql.

Reply:
In Postgres, `user` is a reserved SQL keyword. You should avoid naming your tables using reserved keywords. Keyword usage was not expected to be fixed in the scope of this PR (just bindvars usage).

In tests, I have just updated queries for demonstrating `sql.Rebind` actually called and bindvars type changed.

Reply:
I'm not sure, but potentially with a table name like `interval` with MySQL, you can get similar errors.

We need a new issue regarding keyword support (for both MySQL and PostgreSQL) in CRUD Handlers.

Reply:
Raised [issue](https://github.com/gofr-dev/gofr/issues/551)